dataservice/api/common/schemas.py

```diff
@@ -81,14 +81,29 @@ class ErrorSchema(Schema):
     """ Handles HTTPException marshalling """
 
     class Meta:
-        fields = ('description', 'code')
+        fields = ("description", "code")
 
     @post_dump(pass_many=False)
     def wrap_envelope(self, data):
         return {'_status': {'message': data['description'],
                             'code': data['code']}}
 
 
+def error_response_generator(resource_name, status_code):
```

Comment:
I'm not sure we should regenerate an identical schema for every error type. It seems to violate the idea of polymorphism, which swagger supports.